WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
holy shit forget that, `drop_while` is the shit. adapting that other guy's solution:

```ruby
solve_times.drop_while { |st| !st.skipped? }.all?(&:skipped)
```